packages/@aws-cdk/aws-codepipeline/lib/action.ts

```diff
@@ -340,6 +408,27 @@ export abstract class Action implements IAction {
     return rule;
   }
 
+  public notifyOn(id: string, options: notifications.NotifyOnEventOptions = {}): notifications.INotificationRule {
+    const rule = new notifications.NotificationRule(this._scope, id, {
+      ...options,
+      source: this._pipeline,
+    });
+    return rule;
+  }
+
+  public notifyOnStateChange(id: string, options?: notifications.NotifyOptions): notifications.INotificationRule {
```

Comment:
Can you explain something to me? I don't see any mention of a particular Action here. Does this mean this event triggers for every Action transition in the pipeline?

Reply:
same as above.